### PR TITLE
Feat/v23 cart screen

### PR DESCRIPTION
**Pull request by ashishiiit01**
• On Cart screen:
  • It moves when I drag it.
  • On Card screen, weird shadow appears for Tips.
  • Empty space after the McDonald's. Above "Any items request".
  • Add space before "Any items request"

### CODE REVIEW

File: src/screens/Cart.tsx [hunk 1]
```diff
@@ -157,24 +157,34 @@ export default function Cart() {
...
-            </Checkbox>
-            <Box width='90%' color='orange.700'>
+            <Box width={42}>
+              <Checkbox
+                colorScheme='orange'
+                borderColor='orange.700'
+                mr={3}
+                value={''}
+              >
+                {''}
```
Comment (theankurkedia) [inline src/screens/Cart.tsx]: Why are we adding this?

File: src/screens/Cart.tsx [hunk 2]
```diff
@@ -157,24 +157,34 @@ export default function Cart() {
...
+          flexWrap={'wrap'}
+        >
           <Box
             bg='orange.50'
             p={4}
             borderRadius={8}
             borderWidth={1.5}
             borderColor='orange.300'
             flexDirection={'row'}
+            width={window.width - 30}
```
Comment (theankurkedia) [inline src/screens/Cart.tsx]: Why are we using window width? Should use 100% or something like that

File: src/screens/Cart.tsx [hunk 3]
```diff
@@ -157,24 +157,34 @@ export default function Cart() {
...
+              <Checkbox
+                colorScheme='orange'
+                borderColor='orange.700'
+                mr={3}
+                value={''}
+              >
+                {''}
+              </Checkbox>
+            </Box>
+            <Box width={window.width - 120} color='orange.700'>
```
Comment (theankurkedia) [inline src/screens/Cart.tsx]: Again window.width

File: src/screens/Cart.tsx (hunk 1 above)
Comment (ashishiiit01) [reply src/screens/Cart.tsx]: It was giving me Warning for line no 184.
I have removed this line and added **accessibilityLabel props in checkbox**
### WARNING. **If you do not provide children, you must specify an accessibilityLabel for accessibility**

For Line 182  _**value={''}**_, Eslint error is coming 
**Property 'value' is missing in checkbox**

**Pull request merged**